packages/typespec-test/test/openai_generic/generated/typespec-ts/review/openai-generic.api.md

```diff
@@ -43,7 +43,8 @@ export interface ChatCompletionFunctionCallOption {
 }
 
 // @public
-export interface ChatCompletionFunctionParameters extends Record<string, any> {
+export interface ChatCompletionFunctionParameters {
+    additionalProperties?: Record<string, any>;
```

Comment:
what is the typespec for this? we should generate explicit value type for additionalProperties bag.

Reply:
I think this is expected, the typespec is `Record<unknown>`.

```ts
model ChatCompletionFunctionParameters is Record<unknown>;
```

https://github.com/MARYGAO/autorest.typescript/blob/support-additional-properties/packages/typespec-test/test/openai_generic/spec/completions/models.tsp#L204